torchvision/prototype/transforms/_deprecated.py

```diff
@@ -14,6 +14,9 @@
 from ._utils import query_chw
 
 
+DType = Union[torch.Tensor, PIL.Image.Image, features._Feature]
```

Comment:
Why was the name `DType`chosen? I think it is quite misleading. If we are going to spread this to multiple files, we should rethink the name.

Reply:
`DType` is already used across the codebase. I was under the impression that you introduced but probably is @vfdev-5.

Reply:
It is me who introduced it